react-app/src/redux/game/actions.js

```diff
@@ -24,6 +32,81 @@ export const updateStatus = status => ({
   status
 });
 
+const updateGameActions = {
+  updateGameSuccess: (data, obj, dispatch) => {
+    const id = data.id;
+    const gameCount = Number(data.gameCount) + 1;
+    let objUpdate = { gameCount };
+    if (obj.winner && obj.winner === USER_PLAYER_MARK) {
+      const points = Number(data.points) + obj.points;
+      const historyPoints = { ...data.historyPoints, [new Date().getTime()]: obj.points };
+      objUpdate = { ...objUpdate, points, historyPoints };
+    }
+    authService.setUser(id, objUpdate);
+    dispatch({
```

Comment:
Create a private action creator that receives `objUpdate`. Also, you can just return this action. No need to dispatch